src/frontend/components/ExpandedSidebar/ExpandedSidebar.js

```diff
@@ -136,20 +138,21 @@ export const ExpandedSidebar = () => {
         <span className="material-icons-outlined sidebar-icon">thumb_up</span>
         <span className="sidebar-option">Liked Videos</span>
       </NavLink>
-      <NavLink
-        to="/playlists"
-        className={({ isActive }) =>
-          isActive
-            ? "expanded-sidebar-link active-link"
-            : "expanded-sidebar-link"
-        }
-        onClick={linkClickHandler}
-      >
-        <span className="material-icons-outlined sidebar-icon">
-          playlist_play
-        </span>
-        <span className="sidebar-option">Playlists</span>
-      </NavLink>
+      {playlists.map((playlist) => (
+        <NavLink
+          key={playlist._id}
+          to={`/playlists/${playlist._id}`}
+          className={({ isActive }) =>
+            isActive
+              ? "expanded-sidebar-link active-link"
+              : "expanded-sidebar-link"
```

Comment:
extract outside and pass only a single word as per active style convention

Reply:
Right! Thank you.